.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
       - develop
 
 concurrency:
-  group: ${ github.workflow }-${ github.head_ref }
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Ah, I didn't notice the single brackets, I looked at this when it seemed like one workflow was cancelling another. :)